src/query/v2/CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+define_add_lcp(add_lcp_query lcp_query_2_cpp_files generated_lcp_query_2_files)
```

Comment:
Can this be `lcp_query_v2`? Just to be consistent how the lcp related things are handled in storage v2/v3.

Reply:
And I just noticed I also didn't update this properly in storage-v3! Note for myself.